test/Microsoft.VisualStudio.Threading.Tests/AsyncReaderWriterResourceLockTests.cs

```diff
@@ -762,6 +762,31 @@ public async Task PreparationReservesLock()
         Assert.True(!preparationStartTask.Result.CanBeCanceled);
     }
 
+    [Fact]
+    public async Task PreparationReservesThrottlingScheduler()
```

Comment:
Did you mean _preserves_?

```suggestion
    public async Task PreparationPreservesThrottlingScheduler()
```